deploy/install-driver.sh

```diff
@@ -35,7 +35,7 @@ fi
 
 echo "Installing NFS CSI driver, version: $ver ..."
 kubectl apply -f $repo/rbac-csi-nfs-controller.yaml
-kubectl apply -f $repo/csi-nfs-driverinfo.yaml
+kubectl apply -f $repo/csi-nfs-driverinfo.yaml --validate=false
```

Comment:
it's not necessary to make it as default, remove this

Reply:
Actually, in our e2e test, this script is being called. And in Azure Stack CI, we are testing against Kubernetes v1.15, so it's necessary for older Kubernete versions, especially on the home page we declare that we support 1.14+. I can add a flag here.

Reply:
In my opinion, it's benign to add this as default, or we can change the home page to say we only support k8s 1.16+. Otherwise, this script will not work for k8s older than 1.16.

Reply:
it's a burden to support 1.15 since 1.15 is already not supported by upstream, there could be new features later that could easily break this. If user still wants to use 1.15, I think provide helm installation is enough, while we cannot guarantee it's fully supported.

Reply:
Ok make sense, so let me change the support version to 1.16+ on home page? Or we leave 1.14+ there and add notes to disable the api validation for older versions?

Reply:
yes, thanks. We don't support old versions officially.